pandas/tests/io/test_sql.py

```diff
@@ -938,6 +938,12 @@ def test_database_uri_string(self):
 
         # using driver that will not be installed on Travis to trigger error
         # in sqlalchemy.create_engine -> test passing of this error to user
```

Comment:
you would need to do

```
pg8000 = pytest.pyimportskip('pg8000')  # noqa
```

you need to explicitly install this though in the requirements-2.7.pip (and other ones if needed)

cc @jorisvandenbossche